examples/pylint/r1736_unnecessary_list_index_lookup.py

```diff
@@ -0,0 +1,4 @@
+colours = ['red', 'blue', 'yellow', 'green']
+
+for index, colour in enumerate(colours):
+    print(colours[index])  # Error on this line
```

Comment:
This is example is good, except the corrected version no longer uses `index`, which is a bit subtle.

So instead, you please add a separate `print(index)` call to the example, so that in the corrected version `index` will still be used.